server_scripts/glassfish_administer_service.py

```diff
@@ -55,11 +57,12 @@ def _parseArgs():
 
 class CommandGlassfishDomain(object):
 
-    def __init__(self, asadminpath, domain, domainpath, verbose):
+    def __init__(self, asadminpath, domain, domainpath, verbose, instanceport):
         self.asadminpath = asadminpath
         self.domain = domain
         self.path = None
         self.verbose = verbose
+        self.instanceport = instanceport
```

Comment:
I don't think it needs to be an instance variable, does it? It's only used in the constructor.

Reply:
fixed.